### PR TITLE
feat: add support for make it mine and deploy via cloudbuild trigger …

### PR DESCRIPTION
**Pull request by vandnagarggoogle**
…for load balanced vms

### CODE REVIEW

File: tutorial.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+<walkthrough-metadata>
+  <meta name="title" content="Edit Jumpstart Solution and deploy tutorial " />
+  <meta name="description" content="Make it mine neos tutorial" />
+  <meta name="component_id" content="1361081" />
+  <meta name="short_id" content="true" />
+</walkthrough-metadata>
+
+# Customize an Load balanced managed VMs Solution
```
Comment (singhalbhaskar) [inline tutorial.md:8]: a Load

**Review by singhalbhaskar — COMMENTED** [review]
LGTM, just one nitpick!

File: tutorial.md (hunk 1 above)
Comment (vandnagarggoogle) [reply tutorial.md]: done

**Pull request merged**